src/main/scala/chiseltest/package.scala

```diff
@@ -57,6 +57,29 @@ package object chiseltest {
     }
   }
 
+  private object BitsDecoders {
+    import chisel3.internal.firrtl.{BinaryPoint, KnownBinaryPoint, UnknownBinaryPoint}
+
+    def boolBitsToString(bits: BigInt): String = (bits != 0).toString
+
+    def fixedToString(binaryPoint: BinaryPoint): BigInt => String = {
+      def inner(bits: BigInt): String = {
+        binaryPoint match {
+          case KnownBinaryPoint(binaryPoint) =>
+            val bpInteger = 1 << binaryPoint
+            (bits.toFloat / bpInteger).toString
+          case UnknownBinaryPoint => "[unknown binary point]"
+        }
+      }
+      inner
+    }
+
+    def enumToString(record: EnumType): BigInt => String = {
+      def inner(bits: BigInt): String = "[unimplemented enum decode]"
```

Comment:
ChiselGUI gets the names of enum values, see EnumManager in that repo, it seems to be picking them up 
by collecting the `EnumDefAnnotation` and the EnumComponentAnnotation`

Reply:
Good to know, but I think (unless there's like overwhelming demand for this) I'll wait for underlying enum reflection-like support in base Chisel, instead of adding more complexity while only addressing part of the problem (I don't think enum annotations can address #111, at least not without more hacks?)